canvas_syllabus_export/settings/aws.py

```diff
@@ -135,8 +132,89 @@
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
-STATIC_ROOT = '/home/ubuntu/static'
+STATIC_ROOT = os.path.normpath(os.path.join(BASE_DIR, 'http_static'))
 STATIC_URL = '/static/'
-STATICFILES_DIRS = ('/home/ubuntu/canvas-syllabus-export/canvas_syllabus_export/syllabuspdf/static', )
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'syllabusexport', 'syllabuspdf', 'static'),
```

Comment:
Shouldn't this be:

 ```python
os.path.join(BASE_DIR, 'syllabuspdf', 'static')
```

Since that is where the static files are located for the "syllabuspdf" app? You should be able to test this using the "findstatic" command:

```
$ ./manage.py findstatic syllabuspdf/css/styles.css
```

And then running the "collectstatic" command should copy all the static files to the "http_static" folder in the base directory.

```
./manage.py collectstatic
```